builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -139,40 +145,391 @@ class IntValidator implements TypeValidator {
   }
 }
 
+class StringValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof String)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class BooleanValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof boolean)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class EnumValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof Enum)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class ListValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof List)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class MapValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof Map)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class JsonValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof JSONObject)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class SequenceValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      if(!aValue instanceof List)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class IamArnValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      def pattern = "^arn:aws:iam::\\d{12}:role/?[a-zA-Z_0-9+=,.@\\-_/]+"
+      def match = aValue ==~ pattern
+      if(!match)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
+      throw new IllegalArgumentException(aValue);
+    }
+  }
+}
+
+class KmsArnValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    try {
+      def pattern = "^arn:aws:kms::\\d{12}:key/?[a-zA-Z_0-9+=,.@\\-_/]+"
+      def match = aValue ==~ pattern
+      if(!match)
+      throw new IllegalArgumentException(aValue);
+    } catch(e) {
```

Comment:
Do we really need catch(e) {...} in this situation. And if we do then it would be a good idea to nest it into the wrapping exception

Reply:
Removed the unwanted try catch